app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CommentsController < ApplicationController
+  before_action :set_project
+  before_action :set_comment, only: [:edit, :update, :destroy]
+  before_action :validate_user, only: [:edit, :update, :destroy]
+  before_action :set_user, only: [:new, :edit]
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = @project.comments.new(comment_params)
+    @comment.user = current_user
+    @comment.save
+  end
+
+  def edit
+  end
+
+  def update
+    @comment.update(comment_params)
+  end
+
+  def destroy
+    @comment.destroy
+    redirect_to project_path(@project)
```

Comment:
app/views/comments/destroy.js.erb

we have this file and we are also redirecting? Do you see the issue?